### DISCUSSION TITLE
This web browser does not support Web Bluetooth or it is not enabled

### DISCUSSION THREAD

**Discussion by Finnlego** [pybricks/support#687] [Q&A]
Hi Pybricks, I have tried using a number of different browsers, but all give the following error: This web browser does not support Web Bluetooth or it is not enabled.
What browser should I use (I would prefer Crome), and how should it be configured?
Thanks in advance.

**Comment by dlech** [comment]
Google Chrome works - except on iOS. If you are on Linux, you need to enable experimental web features in the browser. Also, the page needs to be using https (the server should automatically redirect if it isn't, so I don't expect this to be the problem).

**Comment by laurensvalk** [comment]
Does your computer have built-in BLE, or do you have an external Bluetooth dongle that supports BLE (bluetooth low energy)? Make sure bluetooth is enabled in your system settings as well.

**Reply by dlech** [reply]
FYI, missing or disabled Bluetooth hardware gives a different error message, so we can rule that out in this case.

**Comment by Finnlego** [comment]
Thanks a lot for fast response. I installed Chrome on my Mac, and guess
what, it seems to be working! (I could not get it to work on my (old)
Windows PC).
Ready for next step :-)

Den tor. 21. jul. 2022 kl. 19.31 skrev David Lechner <
***@***.***>:
